### PR TITLE
fix(OTSession): clean up OTSession event handlers

### PR DESCRIPTION
**Pull request by julianoddreis**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message

### Solves issue(s)

https://github.com/opentok/opentok-react-native/issues/271

### CODE REVIEW

**Comment by nicoladj77** [comment]
This PR works, I had a complex issues realted to event handlers firing twice and this solved it. Is there any reason not to merg this into master?

File: src/OT.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ const removeNativeEvents = (events) => {
   const eventNames = Object.keys(events);
   OT.removeNativeEvents(eventNames);
   each(events, (eventHandler, eventType) => {
+    nativeEvents.removeAllListeners(eventType);
     nativeEvents.removeListener(eventType, eventHandler);
```
Comment (marsch) [inline src/OT.js:47]: from my perspective the call to `removeAllListeners` just should happen once per `eventType` - nevertheless the call on the next line is obsolete, isn't it?

**Comment by Alan-Billi** [comment]
Any news on merging this ?
I've done the same update on my local code and it's fixing the duplicated instances of the component.

I've removed the old line 
//nativeEvents.removeListener(eventType, eventHandler);

File: src/OT.js (hunk 1 above)
Comment (enricop89) [reply src/OT.js:47]: I agree, we should remove line 47. @abdulajet any thoughts?

**Review by enricop89 — APPROVED** [review]
LGTM

**Pull request merged**